interface/src/avatar/MyAvatar.h

```diff
@@ -685,14 +687,21 @@ public slots:
     bool _clearOverlayWhenMoving { true };
 
     const float ROLL_CONTROL_DEAD_ZONE_DEFAULT = 8.0f; // deg
+#ifdef USE_SIMPLE_ROLL_CONTROL
+    const float ROLL_CONTROL_SPEED_DEFAULT = 180.0f; // deg/sec
```

Comment:
Note: the alternative ROLL_CONTROL_SPEED_DEFAULT is a bit simpler to understand and tune: it is just a max angular velocity (deg/sec).  I didn't spend too much time tuning it just right to match the old behavior.  This value is close but might need a little bit of adjustment.  It currently sits as 180 deg/sec which is 0.5 revolution/sec.